.github/workflows/experiment.yml

```diff
@@ -0,0 +1,137 @@
+name: Self-hosted runner (scheduled)
+
+on:
+  push:
+    branches:
+      - ci_*
```

Comment:
I think this needs to be modified to `stas00:*` or something that matches your branch.

Reply:
there is no `username` here, just the branch name. 

To fix the integration job, I had to change `github-torch-hub.yml` for it to use my username when it tries to download a zip of the branch code - and then this job finally works, but it is of no use to what we are trying here.